src/screens/UnlockScreen.tsx

```diff
@@ -64,22 +115,50 @@ export class UnlockScreen extends PureComponent<Props, State> {
     });
   };
 
+  onCountFinish = () => {
+    this.setState({ failedTimes: 0, isCount: false });
+  };
+
+  isTimeCounterVisible = () => {
+    return dayjs().unix() < this.props.timeCounter.timestamp && this.state.isCount;
+  };
+
   render() {
     const { error, pin } = this.state;
-    return (
-      <View style={styles.container}>
-        <StatusBar barStyle="dark-content" />
-        <View style={styles.imageContainer}>
-          <Image source={images.portraitLogo} style={styles.logo} resizeMode="contain" />
+    if (this.isTimeCounterVisible()) {
+      NavigationService.navigate(Route.TimeCounter, {
+        onCountFinish: this.onCountFinish,
+        timestamp: this.props.timeCounter.timestamp,
+      });
```

Comment:
return null here and you don't need the condition `!this.isTimeCounterVisible()`